tests/app/authentication/test_jwt.py

```diff
@@ -0,0 +1,285 @@
+import unittest
+from tests.app.authentication import SR_PRIVATE_PEM
+from app.authentication.jwt_decoder import Decoder
+from app.authentication.invalid_token_exception import InvalidTokenException
+import base64
```

Comment:
Generally speaking Python library imports normally come first before app specific ones (thought this was a PEP8/Lint thing?) I could be wrong...